<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SCAPES System Menu</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Program Directory</string>
    </property>
   </widget>
   <widget class="QListView" name="ProgramList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>421</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="CreateButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Program</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CompileButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>200</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Compile Program</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RunButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>250</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Run Program</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>300</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Program</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="BottomText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>591</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>100</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Program</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CloseButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>150</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Close Program</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="TextEditor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>421</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdmin_Options"/>
   <addaction name="actionChange_Language"/>
   <addaction name="actionChange_Directory"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdmin_Options">
   <property name="text">
    <string>DEBUG: Change User Type</string>
   </property>
  </action>
  <action name="actionChange_Language">
   <property name="text">
    <string>Change Language</string>
   </property>
  </action>
  <action name="actionChange_Directory">
   <property name="text">
    <string>Change Directory</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
